mesa/batchrunner.py

```diff
@@ -159,11 +188,9 @@ def run_all(self):
                     pbar.update()
 
     def run_iteration(self, kwargs, param_values, run_count):
-        kwargscopy = copy.deepcopy(kwargs)
-        model = self.model_cls(**kwargscopy)
-        self.run_model(model)
-
-        # Collect and store results:
+        kwargs_copy = copy.deepcopy(kwargs)
```

Comment:
I think we don't need to copy the kwargs here. They are just used to instantiate the model and are not modified. Or am I overlooking something??

Reply:
I want to say I thought too and tried to remove it but got an error. I will try it again and see what the error is

Reply:
Maybe it is indeed necessary, but if it is not to much of a hassle maybe you can give it another shot

Reply:
Will do

Reply:
Does not need the copy, must be a remnant of when I was trying to keep the MP and non-MP in the same function...removed.